<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0" alignment="Qt::AlignVCenter">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>状态</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2" colspan="2" alignment="Qt::AlignTop">
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>300</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2" alignment="Qt::AlignTop">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>设备状态</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:green</string>
         </property>
         <property name="text">
          <string>未连接</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" rowspan="2" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>当前卡本轮呼叫数量</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" rowspan="2" alignment="Qt::AlignTop">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>当前正在使用SIM卡</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" alignment="Qt::AlignVCenter">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>线路账号 102333@10.8.9.9:5060</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2" alignment="Qt::AlignTop">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>SIM卡连续呼叫数量</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCommandLinkButton" name="commandLinkButton">
         <property name="text">
          <string>未注册</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" alignment="Qt::AlignTop">
        <widget class="QComboBox" name="comboBox_sndPlay">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="minimumContentsLength">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="6" column="2" alignment="Qt::AlignTop">
        <widget class="QComboBox" name="comboBox_sndMic"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>声卡设备 麦克风</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>声卡设备 播放</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButtonSIP">
      <property name="text">
       <string>线路账号</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actiongaojie"/>
    <addaction name="actionTestcall"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionyantong"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiongaojie">
   <property name="text">
    <string>显示日志</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionyantong">
   <property name="text">
    <string>言通</string>
   </property>
  </action>
  <action name="actionTestcall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>测试呼叫</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonSIP</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SIPAccount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_actiongaojie_triggered()</slot>
  <slot>on_actionyantong_triggered()</slot>
  <slot>SIPAccount()</slot>
  <slot>ChangCard()</slot>
 </slots>
</ui>
